astrocut/cutouts.py

```diff
@@ -184,7 +184,7 @@ def _parse_extensions(infile_exts, infile_name, user_exts):
                     
 def fits_cut(input_files, coordinates, cutout_size, correct_wcs=False, extension=None, 
              single_outfile=True, cutout_prefix="cutout", output_dir='.',
-             memory_only=False, verbose=False):
+             memory_only=False, verbose=False, fsspec_kwargs={"anon": True}):
```

Comment:
It's more user-friendly for `fsspec_kwargs` to be automatically built within the method when it's called, rather than fed as a keyword argument.